### PR TITLE
doc news: add news for 13.0.1

### PR DESCRIPTION
**Pull request by HashidaTKS**
TODO

* [x] Translation

### CODE REVIEW

**Comment by HashidaTKS** [comment]
@kou @komainu8 

Would you review this?

**Comment by kou** [comment]
Sorry. I don't have enough time to review this for now...

**Comment by HashidaTKS** [comment]
> Sorry. I don't have enough time to review this for now...

Sorry for bothering you. I've got it.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 1]
```diff
@@ -362,12 +566,12 @@ msgid ""
...
-"tritonn OR \"groonga mysql\"`` と展開して欲しくなりますが、今までは、そのよう"
-"なケースでは定義を追加する必要がありました。したがって、対象のキーワードに同"
-"義語がたくさんあると、同じような定義をたくさん定義する必要があるため、同義語"
-"の定義が煩雑でした。"
+"多くのケースでは、 ``mroonga`` を ``mroonga OR tritonn OR \"groonga "
+"mysql\"`` と展開した場合、 ``tritonn`` や ``\"groonga mysql\"`` も ``mroonga "
+"OR tritonn OR \"groonga mysql\"`` と展開して欲しくなりますが、今までは、その"
+"ようなケースでは定義を追加する必要がありました。したがって、対象のキーワード"
+"に同義語がたくさんあると、同じような定義をたくさん定義する必要があるため、同"
+"義語の定義が煩雑でした。"
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:574]: This modification does not related release note for 13.0.1.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 2]
```diff
@@ -399,8 +603,8 @@ msgid ""
 "synonyms, we just remove ``{\"_key\": \"groonga mysql\": \"representative\": "
 "\"mroonga\"}``."
 msgstr ""
-"同義語を削除する際も、対象のレコードを削除するだけです。例えば、同義語から ``"
-"\"groonga mysql\"`` を削除する場合は、 ``{\"_key\": \"groonga mysql\": "
+"同義語を削除する際も、対象のレコードを削除するだけです。例えば、同義語から "
+"``\"groonga mysql\"`` を削除する場合は、 ``{\"_key\": \"groonga mysql\": "
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:607]: ditto.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 3]
```diff
@@ -967,11 +1168,11 @@ msgid ""
...
-"要素のみを ``--query`` の対象にしています。この例では、 ``[\"I like Groonga"
-"\", \"Use Groonga with Ruby\"]`` は第2要素 ``Use Groonga with Ruby`` に "
-"``Ruby`` を含むのでヒットしていますが、 ``[\"I like Ruby\", \"Use Groonga"
-"\"]`` は第2要素 ``Use Groonga`` に ``Ruby`` を含まないのでヒットしていませ"
-"ん。"
+"要素のみを ``--query`` の対象にしています。この例では、 ``[\"I like "
+"Groonga\", \"Use Groonga with Ruby\"]`` は第2要素 ``Use Groonga with Ruby`` "
+"に ``Ruby`` を含むのでヒットしていますが、 ``[\"I like Ruby\", \"Use "
+"Groonga\"]`` は第2要素 ``Use Groonga`` に ``Ruby`` を含まないのでヒットしてい"
+"ません。"
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:1175]: ditto

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 4]
```diff
@@ -1441,8 +1642,8 @@ msgid ""
 "Groonga returned ``c`` as a result. This was because we normalized the input "
 "``a`` to ``b`` by definitions of ``ColumnNormalizations``, and after that, "
 "we normalized the normalized ``b`` again and it was normalized to ``c``. As "
-"a result, the input ``a`` was converted to ``c`` and matched to ``{\"content"
-"\":\"c\"}`` of the ``Memos`` table."
+"a result, the input ``a`` was converted to ``c`` and matched to "
+"``{\"content\":\"c\"}`` of the ``Memos`` table."
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:1646]: ditto.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 5]
```diff
@@ -1534,8 +1735,9 @@ msgid ""
 "When the number of results of the OR search is many and a query has many OR "
 "conditions, Groonga may response slow with the \"request_cancel\" command."
 msgstr ""
-"OR検索の結果が多い場合や多くのOR条件をもつクエリー場合に、 \"request_cancel"
-"\" コマンドが原因でGroongaのレスポンスが遅くなることがあります。"
+"OR検索の結果が多い場合や多くのOR条件をもつクエリー場合に、 "
+"\"request_cancel\" コマンドが原因でGroongaのレスポンスが遅くなることがありま"
+"す。"
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:1740]: ditto.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 6]
```diff
@@ -2183,8 +2385,8 @@ msgstr ""
 msgid ""
 "``delimiter_regexp`` option is useful if we want to only extract the text of "
 "the same sentence as search keywords. For example, we can use ``\\.\\s*`` to "
-"extract only text in the target sentence. Note that you need to escape ``"
-"\\`` in string."
+"extract only text in the target sentence. Note that you need to escape "
+"``\\`` in string."
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:2389]: ditto.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 7]
```diff
@@ -2649,8 +2851,8 @@ msgid ""
 "than ``'*NP\"...\" OR *NP\"...\" OR ...'``."
 msgstr ""
 "この機能によって、上記の式を ``*NPP\"(a b) (1 2 3) (x y z)\"`` と書くことがで"
-"きます。加えて、 ``*NPP\"(a b) (1 2 3) (x y z)\"`` は、 ``'*NP\"...\" OR *NP"
-"\"...\" OR ...'`` より高速です。"
+"きます。加えて、 ``*NPP\"(a b) (1 2 3) (x y z)\"`` は、 ``'*NP\"...\" OR "
+"*NP\"...\" OR ...'`` より高速です。"
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:2855]: ditto.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 8]
```diff
@@ -3156,8 +3358,8 @@ msgid ""
 "Even if we use ``query_parallel_or(\"tags || tags2\", \"beginner man\")``, "
 "the above record is match by this modification."
 msgstr ""
-"今回の変更によって、 ``query_parallel_or(\"tags || tags2\", \"beginner man"
-"\")`` を使った場合であっても、上記のレコードがマッチするようになりました。"
+"今回の変更によって、 ``query_parallel_or(\"tags || tags2\", \"beginner "
+"man\")`` を使った場合であっても、上記のレコードがマッチするようになりました。"
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:3362]: ditto.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 9]
```diff
@@ -3589,11 +3791,11 @@ msgstr ""
 
 msgid ""
 "In particular, the following query adjust the score of match records by the "
-"following condition(``'content @ \"Groonga\" *~2.5 content @ \"Mroonga"
-"\")'`` )."
+"following condition(``'content @ \"Groonga\" *~2.5 content @ "
+"\"Mroonga\")'`` )."
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:3795]: ditto.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 10]
```diff
@@ -3589,11 +3791,11 @@ msgstr ""
...
 msgstr ""
-"具体的には、 以下の条件( ``'content @ \"Groonga\" *~2.5 content @ \"MySQL"
-"\"'`` ) にマッチしたレコードのスコアーを調整します。"
+"具体的には、 以下の条件( ``'content @ \"Groonga\" *~2.5 content @ "
+"\"MySQL\"'`` ) にマッチしたレコードのスコアーを調整します。"
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:3798]: ditto.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 11]
```diff
@@ -3626,9 +3828,9 @@ msgstr ""
 "ルトの重みは 0.5です。 ``*>`` のデフォルトの重みは 2.0 です。"
 
 msgid ""
-"We can specify score quantity by ``*<${score_quantity}`` and ``*>"
-"${score_quantity}``. Then, if we specify ``*<${score_quantity}``, a plus or "
-"minus sign of ``${score_quantity}`` is reverse."
+"We can specify score quantity by ``*<${score_quantity}`` and "
+"``*>${score_quantity}``. Then, if we specify ``*<${score_quantity}``, a plus "
+"or minus sign of ``${score_quantity}`` is reverse."
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:3833]: ditto.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 12]
```diff
@@ -3942,8 +4144,9 @@ msgstr ""
 
 msgid ""
 "We can specify multiple token filters with options like ``--token_filters "
-"'TokenFilterStopWord(\"column\", \"ignore\"), TokenFilterNFKC130(\"unify_kana"
-"\", true)'``. [Github#mroonga/mroonga#399][Reported by MASUDA Kazuhiro]"
+"'TokenFilterStopWord(\"column\", \"ignore\"), "
+"TokenFilterNFKC130(\"unify_kana\", true)'``. [Github#mroonga/mroonga#399]"
+"[Reported by MASUDA Kazuhiro]"
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:4149]: ditto.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 13]
```diff
@@ -4051,22 +4254,22 @@ msgstr ""
...
 
 msgid ""
-"We can customize adjustment of weight every key word by only specifying ``<"
-"${WEIGHT}`` or ``>${WEIGHT}`` to target keywords since this release. For "
-"example, \"A\"'s weight is 1 and \"B\"'s weight is 0.9 in ``A <0.1B`` (\"B"
-"\"'s weight decrement 0.1)."
+"We can customize adjustment of weight every key word by only specifying "
+"``<${WEIGHT}`` or ``>${WEIGHT}`` to target keywords since this release. For "
+"example, \"A\"'s weight is 1 and \"B\"'s weight is 0.9 in ``A <0.1B`` "
+"(\"B\"'s weight decrement 0.1)."
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:4260]: ditto.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 14]
```diff
@@ -4051,22 +4254,22 @@ msgstr ""
...
 msgstr ""
 "今回のリリースから、対象のキーワードに ``<${WEIGHT}`` または ``>${WEIGHT}`` "
 "と指定するだけで、キーワードごとに重みの調整量をカスタマイズできます。例え"
-"ば、 ``A <0.1B`` では、 \"A\" の重みは1、 \"B\" の重みは0.9になります。(\"B"
-"\" の重みを0.1デクリメントしています。)"
+"ば、 ``A <0.1B`` では、 \"A\" の重みは1、 \"B\" の重みは0.9になります。"
+"(\"B\" の重みを0.1デクリメントしています。)"
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:4265]: ditto.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 15]
```diff
@@ -4051,22 +4254,22 @@ msgstr ""
...
 
 msgid ""
 "However, note that these forms ( ``>${WEIGHT}...``, ``<${WEIGHT}...``, and "
 "``~${WEIGHT}...`` ) are incompatible."
 msgstr ""
-"ただし、これらの形式(``>${WEIGHT}...``, ``<${WEIGHT}...``, and ``~"
-"${WEIGHT}...``)は互換性が無いことに注意してください。"
+"ただし、これらの形式(``>${WEIGHT}...``, ``<${WEIGHT}...``, and "
+"``~${WEIGHT}...``)は互換性が無いことに注意してください。"
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:4272]: ditto.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 16]
```diff
@@ -4090,17 +4293,17 @@ msgid ""
 "value like ``index_column.xxx``. For example, A value of ``--columns[tags]."
 "value purchases.tag`` was ``[\"apple\",[\"many\"]],[\"banana\",[\"man\"]],"
 "[\"cacao\",[\"man\"]]`` in the following example. In this case, the expected "
-"values was ``[\"apple\",[\"man\",\"many\"]],[\"banana\",[\"man\"]],[\"cacao"
-"\",[\"woman\"]]``. In this release, we can get a correct reference "
+"values was ``[\"apple\",[\"man\",\"many\"]],[\"banana\",[\"man\"]],"
+"[\"cacao\",[\"woman\"]]``. In this release, we can get a correct reference "
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:4297]: ditto.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 17]
```diff
@@ -4090,17 +4293,17 @@ msgid ""
...
 "今までは、 出力値に ``index_column.xxx`` のように指定すると、Groongaは意図し"
 "ない値を返していました。例えば以下の例では、 ``--columns[tags].value "
 "purchases.tag`` の値は、 ``[\"apple\",[\"many\"]],[\"banana\",[\"man\"]],"
 "[\"cacao\",[\"man\"]]`` になりました。このケースでは、期待される値は、 "
-"``[\"apple\",[\"man\",\"many\"]],[\"banana\",[\"man\"]],[\"cacao\",[\"woman"
-"\"]]`` でした。今回のリリースから、以下のように、インデックスカラム経由で参照"
-"先の正しい値を取得できます。"
+"``[\"apple\",[\"man\",\"many\"]],[\"banana\",[\"man\"]],[\"cacao\","
+"[\"woman\"]]`` でした。今回のリリースから、以下のように、インデックスカラム経"
+"由で参照先の正しい値を取得できます。"
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:4306]: ditto.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 18]
```diff
@@ -4131,8 +4334,8 @@ msgstr ""
 "コードをロードできるようにしました。"
 
 msgid ""
-"For example, we can load data like ``\"key\" : \"[ { \"key\" : \"value"
-"\", ..., \"key\" : \"value\" } ]\"`` as below."
+"For example, we can load data like ``\"key\" : \"[ { \"key\" : "
+"\"value\", ..., \"key\" : \"value\" } ]\"`` as below."
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:4338]: ditto.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 19]
```diff
@@ -4311,12 +4514,12 @@ msgid ""
 "The result had been as follows until now. We can see that ``{\"product\": "
 "\"apple\",  \"tag\": \"man\"}`` is not output."
 msgstr ""
-"今までの結果は以下のようになっていました。 ``{\"product\": \"apple\",  \"tag"
-"\": \"man\"}`` が出力されていないことがわかります。"
+"今までの結果は以下のようになっていました。 ``{\"product\": \"apple\",  "
+"\"tag\": \"man\"}`` が出力されていないことがわかります。"
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:4518]: ditto.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 20]
```diff
@@ -4311,12 +4514,12 @@ msgid ""
...
 
 msgid ""
-"The result will be as follows from this release. We can see that ``{\"product"
-"\": \"apple\",  \"tag\": \"man\"}`` is output."
+"The result will be as follows from this release. We can see that "
+"``{\"product\": \"apple\",  \"tag\": \"man\"}`` is output."
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:4522]: ditto.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 21]
```diff
@@ -7155,8 +7358,8 @@ msgstr ""
 "を追加しました。"
 
 msgid ""
-"[:doc:`reference/executables/groonga-httpd`] Added support for ``"
-"$request_time`` in log."
+"[:doc:`reference/executables/groonga-httpd`] Added support for "
+"``$request_time`` in log."
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:7362]: ditto.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 22]
```diff
@@ -7176,8 +7379,8 @@ msgstr ""
 "しい時間を出力します。"
 
 msgid ""
-"[:doc:`reference/executables/groonga-httpd`] Added how to set the ``"
-"$request_time`` in the document."
+"[:doc:`reference/executables/groonga-httpd`] Added how to set the "
+"``$request_time`` in the document."
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:7383]: ditto.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 23]
```diff
@@ -9350,13 +9553,13 @@ msgid ""
 "IDEOGRAPH-8868\" in CP932, the base path detection is buggy."
 msgstr ""
 "[windows] base path検出時の問題を修正しました。システムロケールのDLLパスが、"
-"CP932での「U+8868 CJK UNIFIED IDEOGRAPH-8868」のように0x5c （ASCIIでの ``"
-"\\`` )を含む場合に問題がありました。"
+"CP932での「U+8868 CJK UNIFIED IDEOGRAPH-8868」のように0x5c （ASCIIでの "
+"``\\`` )を含む場合に問題がありました。"
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:9557]: ditto.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 24]
```diff
@@ -9350,13 +9553,13 @@ msgid ""
...
 
 msgid ""
 "[:doc:`/reference/tokenizers`][TokenNgram] Fixed wrong first character "
-"length. It's caused for \"PARENTHESIZED IDEOGRAPH\" characters such as \"U"
-"+3231 PARENTHESIZED IDEOGRAPH STOCK\"."
+"length. It's caused for \"PARENTHESIZED IDEOGRAPH\" characters such as "
+"\"U+3231 PARENTHESIZED IDEOGRAPH STOCK\"."
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:9562]: ditto.

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 25]
```diff
@@ -10462,8 +10665,8 @@ msgid "Release 7.0.6 - 2017-08-29"
 msgstr "7.0.6リリース - 2017-08-29"
 
 msgid ""
-"Supported prefix match search using multiple indexes. (e.g. ``--query \"Foo*"
-"\" --match_columns \"TITLE_INDEX_COLUMN||BODY_INDEX_COLUMN\"``)."
+"Supported prefix match search using multiple indexes. (e.g. ``--query "
+"\"Foo*\" --match_columns \"TITLE_INDEX_COLUMN||BODY_INDEX_COLUMN\"``)."
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:10669]: ditto.

File: doc/source/news.rst [hunk 26]
```diff
@@ -5,6 +5,408 @@
...
+       table_create \
+         --name idx_animals_sound \
+         --flags TABLE_PAT_KEY \
+         --key_type ShortText \
+         --default_tokenizer TokenBigram \
+         --normalizer 'NormalizerNFKC150("unify_kana_prolonged_sound_mark", true)'
+       column_create --table idx_animals_sound --name animals_sound --flags COLUMN_INDEX|WITH_POSITION --type Animals --source sound
+
+       select --table Animals --query sound:@メエメエ
+       #[[0,1677829950.652696,0.01971983909606934],[[[1],[["_id","UInt32"],["_key","ShortText"],["name","ShortText"],["sound","ShortText"]],[1,"1","羊","メーメー"]]]]
```
Comment (komainu8) [inline doc/source/news.rst:89]:
```suggestion
       #[
           [
             0,
             1677829950.652696,
             0.01971983909606934
           ],
           [
             [
               [
                 1
               ],
               [
                 [
                   "_id",
                   "UInt32"
                 ],
                 [
                   "_key",
                   "ShortText"
                 ],
                 [
                   "name",
                   "ShortText"
                 ],
                 [
                   "sound",
                   "ShortText"
                 ]
               ],
               [
                 1,
                 "1",
                 "羊",
                 "メーメー"
               ]
             ]
           ]
         ]
```

File: doc/source/news.rst [hunk 27]
```diff
@@ -5,6 +5,408 @@
...
+       table_create \
+         --name idx_animals_sound \
+         --flags TABLE_PAT_KEY \
+         --key_type ShortText \
+         --default_tokenizer TokenBigram \
+         --normalizer 'NormalizerNFKC150("unify_kana_hyphen", true)'
+       column_create --table idx_animals_sound --name animals_sound --flags COLUMN_INDEX|WITH_POSITION --type Animals --source sound
+
+       select --table Animals --query sound:@メエメエ
+       #[[0,1677829950.652696,0.01971983909606934],[[[1],[["_id","UInt32"],["_key","ShortText"],["name","ShortText"],["sound","ShortText"]],[1,"1","羊","メ-メ-"]]]]
```
Comment (komainu8) [inline doc/source/news.rst:162]:
```suggestion
       #[
           [
             0,
             1677829950.652696,
             0.01971983909606934
           ],
           [
             [
               [
                 1
               ],
               [
                 [
                   "_id",
                   "UInt32"
                 ],
                 [
                   "_key",
                   "ShortText"
                 ],
                 [
                   "name",
                   "ShortText"
                 ],
                 [
                   "sound",
                   "ShortText"
                 ]
               ],
               [
                 1,
                 "1",
                 "羊",
                 "メ-メ-"
               ]
             ]
           ]
         ]
```

File: doc/locale/ja/LC_MESSAGES/news.po (hunk 1 above)
Comment (HashidaTKS) [reply doc/locale/ja/LC_MESSAGES/news.po:574]: I'm thinking about diffs that automatically created by `make`...
I will revert these unrelated changes, but... should they updated?
Could we update these in another pull request?

File: doc/source/news.rst [hunk 28]
```diff
@@ -5,6 +5,408 @@
...
+  1. Use ``*ONP`` with ``max_element_invervals``
+  2. The number of tokens in the matched left phrase is greater than or equal to the number of ``max_element_invervals`` elements.
+
+  Here is an example of this bug.
+
+  .. code-block::
+
+     table_create Entries TABLE_NO_KEY
+     column_create Entries content COLUMN_SCALAR Text
+     table_create Terms TABLE_PAT_KEY ShortText   --default_tokenizer 'TokenNgram("unify_alphabet", false,                                   "unify_digit", false)'   --normalizer NormalizerNFKC121
```
Comment (komainu8) [inline doc/source/news.rst:280]:
```suggestion
     table_create Terms TABLE_PAT_KEY ShortText \
       --default_tokenizer 'TokenNgram("unify_alphabet", false, \
                                                           "unify_digit", false)' \
       --normalizer NormalizerNFKC150
```

File: doc/source/news.rst [hunk 29]
```diff
@@ -5,6 +5,408 @@
...
+
+  .. code-block::
+
+     table_create Entries TABLE_NO_KEY
+     column_create Entries content COLUMN_SCALAR Text
+ 
+     table_create Terms TABLE_PAT_KEY ShortText \
+       --default_tokenizer 'TokenNgram("unify_alphabet", false, \
+                                       "unify_digit", false)' \
+       --normalizer NormalizerNFKC121
```
Comment (komainu8) [inline doc/source/news.rst:378]:
```suggestion
       --normalizer NormalizerNFKC150
```

File: doc/source/news.rst [hunk 30]
```diff
@@ -5,6 +5,408 @@
...
+     load --table Entries
+     [
+     {"content": "abc123456789defg"},
+     {"content": "dededede"}
+     ]
+
+     select Entries \
+       --filter 'content *NP10,1"abc defg$"' \
+       --output_columns '_score, content'
+     #[[0,0.0,0.0],[[[0],[["_score","Int32"],["content","Text"]]]]]
```
Comment (komainu8) [inline doc/source/news.rst:390]:
```suggestion
     #[
         [
           0,
           0.0,
           0.0
         ],
         [
           [
             [
               0
             ],
             [
               [
                 "_score",
                 "Int32"
               ],
               [
                 "content",
                 "Text"
               ]
             ]
           ]
         ]
       ]
```

File: doc/locale/ja/LC_MESSAGES/news.po [hunk 31]
```diff
@@ -69,6 +69,213 @@ msgstr "タグ"
...
+
+msgid "Here is an example of ``unify_kana_hyphen``."
+msgstr "以下は、 ``unify_kana_hyphen`` オプションの使用例です。"
+
+msgid ""
+"[:ref:`query-syntax-near-search-condition`][:ref:`script-syntax-near-search-"
+"operator`] Added a new option ``${MIN_INTERVAL}`` for near-search family."
+msgstr ""
+"[:ref:`query-syntax-near-search-condition`][:ref:`script-syntax-near-search-"
+"operator`] 各近傍検索に ``${MIN_INTERVAL}`` オプションを追加しました。"
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/news.po:98]: この機能を追加したことによって、解決できるケースを記載して方が良いかと思いました。
「オーバラップしているフレーズをスキップできるようになるので、近傍検索の検索結果のノイズを減らすことができます。」とか。

実行例も↑のケースを実際に解決している例にすると、よりメリットが伝わりやすいと思います。

**Review by komainu8 — COMMENTED** [review]
Could you confirm my comments?

File: doc/source/news.rst (hunk 27 above)
Comment (HashidaTKS) [reply doc/source/news.rst]: Fixed.

File: doc/locale/ja/LC_MESSAGES/news.po (hunk 24 above)
Comment (HashidaTKS) [reply doc/locale/ja/LC_MESSAGES/news.po]: Fixed.

File: doc/locale/ja/LC_MESSAGES/news.po (hunk 2 above)
Comment (HashidaTKS) [reply doc/locale/ja/LC_MESSAGES/news.po]: Fixed.

File: doc/locale/ja/LC_MESSAGES/news.po (hunk 1 above)
Comment (HashidaTKS) [reply doc/locale/ja/LC_MESSAGES/news.po]: Revert changes.

File: doc/locale/ja/LC_MESSAGES/news.po (hunk 31 above)
Comment (HashidaTKS) [reply doc/locale/ja/LC_MESSAGES/news.po:98]: 現在の距離の計算方法だと説明が大変だったので省略していたのですが、記載します。

**Comment by HashidaTKS** [comment]
@komainu8 

Thank you, I have addressed your comments.
We are planing to check normalizers, so perhaps we need to add news items.
Please wait to merge (if there is no problem) until we finish to check normalizers.

**Comment by HashidaTKS** [comment]
...or, merge this PR as is, and create a new pull request for adding new news topics if need.

**Comment by komainu8** [comment]
> @komainu8
> 
> Thank you, I have addressed your comments. We are planing to check normalizers, so perhaps we need to add news items. Please wait to merge (if there is no problem) until we finish to check normalizers.

I see.

**Pull request merged**